src/services/generics/generic.setter.service.ts

```diff
@@ -19,6 +20,14 @@ export class GenericSetterService {
         remoteTtl: number,
         localTtl?: number,
     ): Promise<string> {
+        if (typeof value === 'undefined' || value === null) {
+            remoteTtl = CacheTtlInfo.NullValue.remoteTtl;
+            localTtl = CacheTtlInfo.NullValue.localTtl;
+        }
+
+        value = typeof value === 'undefined' ? 'undefined' : value;
```

Comment:
the same duplicated code here. maybe extract an 'util' function

Reply:
done